### ISSUE TITLE
Lyft Line shows "+ null Prime Time"

### ISSUE THREAD

**Issue by kyle-kelly** [kyle-kelly/MMM-lyft#1]
See details [here](https://github.com/kyle-kelly/MMM-uber/issues/2).

**Comment by kyle-kelly** [comment]
The error most likely stems from this [line](https://github.com/kyle-kelly/MMM-lyft/blob/master/MMM-lyft.js#L126). The line should read `if( this.lyftSurge && this.lyftSurge !== "0%" )`.

I'll test this out and release the fix if successful.

**Comment by kyle-kelly** [comment]
Resolved in most recent master commit.

**Comment by seattlecajun** [comment]
Wow, that was fast. Thanks, @kyle-kelly!  I got a notification on my MM that an update was available. :)

I noticed a few times early this am (before your update) that the 'null' issue was intermittent. At one point, it displayed, correctly, a 25% prime time period and then a little later went back to showing 'null'.

I just applied your latest update and will keep an eye on its behavior and report back. Thanks again!
